src/mapml/control/FullscreenButton.js

```diff
@@ -0,0 +1,160 @@
+export var FullscreenButton = L.Control.extend({
```

Comment:
Do you think crediting the original plugin developer is required in this case?

Reply:
I looked at the repo, it's a fork of a leaflet repo, so same license as that. So, I think we give that credit already?